src/peft/tuners/lora/config.py

```diff
@@ -273,6 +273,13 @@ class LoraConfig(PeftConfig):
             parameter when you want to apply LoRA to the ColumnParallelLinear and RowParallelLinear layers of megatron.
         megatron_core (`Optional[str]`):
             The core module from Megatron to use, defaults to `"megatron.core"`.
+        trainable_token_indices (`Optional[Union[List[int], dict[str, List[int]]]]`)
+            Lets you specify which token indices to selectively fine-tune without requiring to re-train the whole
+            embedding matrix using the `peft.TrainableTokensModel` method. You can either specify a list of indices
```

Comment:
> using the `peft.TrainableTokensModel` method.

I wonder if that's important info or can just be dropped.

Reply:
I like pointers like these because I am incentivized to follow up on this without looking at the code and learning something new.

Reply:
Makes sense.